src/main/java/com/xceptance/neodymium/util/NeodymiumConfiguration.java

```diff
@@ -267,4 +267,8 @@ public interface NeodymiumConfiguration extends Mutable
     @Key("neodymium.logNeoVersion")
     @DefaultValue("true")
     public boolean logNeoVersion();
+
+    @Key("neodymium.report.enableStepLinks")
+    @DefaultValue("false")
```

Comment:
Could you switch this to default true? It's nice to be able to deactivate stuff, but no one will dig through the wiki to find new features.